waybar-backlight.5.scd

```diff
@@ -0,0 +1,68 @@
+waybar-backlight(5)
+
+# NAME
+
+waybar - backlight module
+
+# DESCRIPTION
+
+The *backlight* module displays the current backlight level.
+
+# Config
+
+*interval* ++
+	typeof: integer ++
+	default: 2 ++
+	The interval in which information gets polled.
+
+*format* ++
+	typeof: string ++
+	default: {percent}% ++
+	The format, how information should be displayed. On {} data gets inserted.
+
+*max-length* ++
+	typeof: integer ++
+	The maximum length in characters the module should display.
+
+*rotate* ++
+	typeof: integer ++
+	Positive value to rotate the text label.
+
+*states* ++
+	typeof: array ++
+	A number of backlight states which get activated on certain brightness levels.
+
+*on-click* ++
+	typeof: string ++
+	Command to execute when the module is clicked.
+
+*on-click-right* ++
+	typeof: string ++
+	Command to execute when the module is right clicked.
+
+*on-scroll-up* ++
+	typeof: string ++
+	Command to execute when performing a scroll up on the module.
+
+*on-scroll-down* ++
+	typeof: string
+	Command to execute when performing a scroll down on the module.
+
+*smooth-scrolling-threshold* ++
+	typeof: double
+	Threshold to be used when scrolling.
```

Comment:
A tried a different format for representing the tables in the wiki here. I think it's easier to navigate than the tables generated by scdoc and scales better with narrow terminals (which in the case of using a tiling window manager is very important).

I alos find the syntax much easier to read than the table syntax used by scdoc.

Let me know what you think.